central/views/nodecve/view_impl.go

```diff
@@ -32,6 +32,8 @@ func (n *nodeCVECoreViewImpl) Count(ctx context.Context, q *v1.Query) (int, erro
 		return 0, err
 	}
 
+	q = withoutOrphanedCVEsQuery(q)
```

Comment:
There would have been no way for someone who didn't write this view to know that this thing exists because it doesn't follow to usual searching via datastore approach. Need to find a better way to make these views easily searchable.

Reply:
Add a ticket to ROX-24279 with as much information as possible for us to investigate it.

Reply:
Created [ROX-25830](https://issues.redhat.com/browse/ROX-25830)